README.md

```diff
@@ -54,7 +54,11 @@ gem 'json'
 gem 'redis'
 ```
 
-And a worker
+How we and a worker? Firstly create a file and name it as `boot.rb`
```

Comment:
Suggestion for a tweak: first sentence should probably be something like "How do we add a worker?"